arch/X86/X86MappingReg.inc

```diff
@@ -110,14 +110,14 @@
 { X86_FP5, X86_REG_FP5 },
 { X86_FP6, X86_REG_FP6 },
 { X86_FP7, X86_REG_FP7 },
-{ X86_K0, X86_REG_K0 },
-{ X86_K1, X86_REG_K1 },
-{ X86_K2, X86_REG_K2 },
-{ X86_K3, X86_REG_K3 },
-{ X86_K4, X86_REG_K4 },
-{ X86_K5, X86_REG_K5 },
-{ X86_K6, X86_REG_K6 },
-{ X86_K7, X86_REG_K7 },
+{ X86_K0, 0 },
+{ X86_K1, 0 },
+{ X86_K2, 0 },
+{ X86_K3, 0 },
+{ X86_K4, 0 },
+{ X86_K5, 0 },
+{ X86_K6, 0 },
+{ X86_K7, 0 },
```

Comment:
this is something that I fixed a while ago https://github.com/capstone-engine/capstone/pull/1689
We should be careful, some of these differences that we see might be other fixes that people did after the table generation.

Reply:
That's right. I noticed that. I just pushed a fix.